### PR TITLE
Introduce generated codes into codebase

### PR DESCRIPTION
**Pull request by hanahmily**
SWCK intends to implement kubernetes metrics server by leveraging cli's GraphQL relevant packages. There are several generated codes that should be put into codebase in order to compile the cli as a library. The changes include:

 * [x] Add schema.go. packr relevant go files into the codebase.
 * [x] Insert Apache header into above go files.
 * [x] Add a check task into CI to check whether those files are the latest.
 * [x] Update assets generation implement

### CODE REVIEW

File: hack/build-header.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```
Comment (kezhenxu94) [inline hack/build-header.sh:1]: License header

**Comment by hanahmily** [comment]
I found the generated file located at `packr` varies in different environments. After some investigation, I found that 

https://github.com/gobuffalo/packr/blob/4a2800e19d0afeb585c69332dea4320ea26f9254/v2/jam/store/disk.go#L50

Global Key of the disk is based on the absolute path of `path`, that's why the content is not the same with a different path.

This issue seems not fixed without elaboration, and I prefer to use go-bindata instead of packr. The generated go files embed assets without md5 hex generation.

File: hack/build-header.sh (hunk 1 above)
Comment (hanahmily) [reply hack/build-header.sh:1]: done

**Comment by kezhenxu94** [comment]
> I prefer to use go-bindata instead of packr.

It works for me if there is no similar issue by using go-bindata

**Comment by fgksgf** [comment]
FYI, @kezhenxu94 I tested several commands, it works well.

**Review by kezhenxu94 — COMMENTED** [review]
Now we don't need https://github.com/apache/skywalking-cli/blob/master/dist/licenses/LICENSE-packr this anymore. Otherwise LGTM

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**